artemis-server/src/main/java/org/apache/activemq/artemis/core/security/impl/SecurityStoreImpl.java

```diff
@@ -380,7 +380,7 @@ private void authenticationFailed(String user, RemotingConnection connection) th
       ActiveMQServerLogger.LOGGER.securityProblemWhileAuthenticating(e.getMessage());
 
       if (AuditLogger.isResourceLoggingEnabled()) {
-         AuditLogger.userFailedAuthenticationInAudit(null, e.getMessage());
+         AuditLogger.userFailedAuthenticationInAudit(null, e.getMessage(), connection.getID().toString());
```

Comment:
That's a fair question, but it's not really germane to this PR since it hasn't been changed. I recommend you follow-up on the ActiveMQ users list about this.